cogs/custom.py

```diff
@@ -38,14 +42,73 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
             raise BadArgument("Please provide at least 2 options.")
         if len(options) > 20:
             raise BadArgument("I can only handle 20 options!")
-
         codepoint_start = 127462  # represents "regional_indicator_a" unicode value
         options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=codepoint_start)}
         embed = Embed(title=title, description="\n".join(options.values()))
         message = await ctx.send(embed=embed)
         for reaction in options:
             await message.add_reaction(reaction)
 
+    @command()
+    async def members(self, ctx: Context) -> None:
+        """Returns the number of members in a server."""
+        await ctx.send(embed=Embed(title="Member count", description=ctx.guild.member_count, color=Color.dark_purple()))
+
+    @command()
+    async def status(self, ctx: Context, member: Member = None) -> None:
+        """Returns the status of a member."""
+        if member is None:
+            member = ctx.author
+
+        if member.id == constants.owner_id:
+            embed = status_embed(member, description="Not telling")
+        else:
+            embed = status_embed(member)
+
+        await ctx.send(embed=embed)
+
+    @command()
+    async def pfp(self, ctx, member: Member = None) -> None:
+        """Displays the profile picture of a member."""
+        if member is None:
+            message, url = "Your avatar", ctx.author.avatar_url
+        elif member == ctx.me:
+            message, url = "My avatar", member.avatar_url
+        else:
+            message, url = f"{member} avatar", member.avatar_url
+
+        embed = Embed(description=message, color=Color.dark_blue())
+        embed.set_image(url=url)
+        await ctx.send(embed=embed)
+
+    @command(aliases=["git"])
+    async def github(self, ctx: Context) -> None:
+        """GitHub repository"""
+        await ctx.send(
+            embed=Embed(
+                title="Github Repo",
+                description=f"[Click Here]({constants.github_repo_url}) to visit the Open Source Repo of HotWired",
+                color=Color.dark_blue(),
+            )
+        )
+
+    @command()
+    @cooldown(1, 10, BucketType.user)
+    async def countdown(self, ctx: Context, start: int) -> None:
+        try:
+            await ctx.message.delete()
+        except Forbidden:
+            pass
```

Comment:
Use `contextlib.suppress` to suppress these errors, it's usually not a good thing to use `try-except` block when you're only using `pass` there `contextlib.suppress` is designed exactly for this:

```py
from contextlib import suppress


with suppress Forbidden:
    await ctx.message.delete()
```